readme.md

```diff
@@ -654,6 +654,7 @@
 - [Prometheus](https://github.com/roaldnefs/awesome-prometheus#readme) - Open-source monitoring system.
 - [Homematic](https://github.com/homematic-community/awesome-homematic#readme) - Smart home devices.
 - [Ledger](https://github.com/sfischer13/awesome-ledger#readme) - Double-entry accounting on the command-line.
+- [Web Monetization](https://github.com/thomasbnt/awesome-web-monetization) - Web service that allows you to send money directly in your browser. 
```

Comment:
This does not make it clear that it's an open standard. It makes it sound like it's a paid API.

Reply:
Edited with [`f078080` (#1812)](https://github.com/sindresorhus/awesome/pull/1812/commits/f078080f4659e4d95336f46ee65d1f4c263de2f5)